testing/test_motors.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import context
+import unittest
+import cv2
+import roboplot.core.stepper_motors as stepper_motors
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+import time
+
+# noinspection PyUnresolvedReferences
+import test_runner
+
+
+# Each set of tests can be in their own class, but it must derive from unnit.TestCase
+class MotorTest(unittest.TestCase):
+
+    def tearDown(self):
+        GPIO.cleanup()
+
+    # Each test has its own function here. There are lots of different functions to use in tests
+    def testPass(self):
+        self.assertTrue(True)
+
+    def testStepLarge(self):
```

Comment:
This test name isn't that enlightening. I know you are going to test something about the large stepper motor, but I don't know what / how much!

One school of thought writes (concise) sentences describing what you are testing.
If the name you then want to give it is too long then maybe you need to split up the test / test class.
E.g. Here you might have a `unittest.TestCase` called `LargeStepperTest` with a member variable for the large stepper motor. Your existing method could then be named `testStepsGpioPinsClockwiseCorrectly`. You could then have a sister method `testStepsGpioPinsInReverseWhenClockwiseIsFalse` to test the behaviour when the `clockwise` member is `False`.